### PR TITLE
Add resistance effect for Spirit's Mercy

### PR DESCRIPTION
**Pull request by pedrogrullada**
also add damage buttons for Spirit's Absolution and Spirit's Anguist, and add some icons for the recently added Celebrity archetype effects.

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -2569,7 +2569,14 @@
                 "Wit": "The foe takes a -2 circumstance penalty to attack rolls against you until the start of your next turn. @UUID[Compendium.pf2e.feat-effects.Item.cu10OjQUdY0a9gpj]{Effect: Exemplary Finisher (Wit)}"
             },
             "ExorcistDedication": {
-                "SpiritDwelling": "Your Spirit Dwelling contains a spirit"
+                "SpiritDwelling": "Your Spirit Dwelling contains a spirit",
+                "SpiritsMercy": {
+                    "Prompt": "Select the cost of your Spirit's Mercy",
+                    "PromptLevel": "Select the level of your Spirit Remnant",
+                    "ResistanceLabel": "spirits"
```
Comment (TikaelSol) [inline static/lang/re-en.json:2576]: Move this to the custom IWR localizations

Comment (pedrogrullada) [reply static/lang/re-en.json]: Moved!

File: static/lang/re-en.json [hunk 2]
```diff
@@ -613,6 +613,7 @@
                 "PersistentPoison": "persistent poison",
                 "PhysicalRangedAttacks": "physical ranged attacks",
                 "RangedWeaponAttacks": "ranged weapon attacks",
+                "SpiritsMercy": "spirits",
```
Comment (TikaelSol) [inline static/lang/re-en.json:616]:
```suggestion
                "Spirits": "spirits",
```

File: packs/feat-effects/effect-spirits-mercy.json [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+                                    "origin:trait:undead",
+                                    "origin:trait:incorporeal"
+                                ]
+                            },
+                            "origin:trait:haunt"
+                        ]
+                    }
+                ],
+                "key": "Resistance",
+                "label": "PF2E.IWR.Custom.SpiritsMercy",
```
Comment (TikaelSol) [inline packs/feat-effects/effect-spirits-mercy.json:192]:
```suggestion
                "label": "PF2E.IWR.Custom.Spirits",
```

File: packs/feat-effects/effect-spirits-mercy.json [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
```
Comment (TikaelSol) [inline packs/feat-effects/effect-spirits-mercy.json:168]:
```suggestion
                "label": "PF2E.IWR.Custom.Spirits",
```

**Pull request merged**